src/webots/gui/WbX3dStreamingServer.cpp

```diff
@@ -142,19 +142,7 @@ void WbX3dStreamingServer::sendUpdatePackageToClients() {
 
 bool WbX3dStreamingServer::prepareWorld() {
   try {
-    bool regenerationRequired = mX3dWorldReferenceFile != WbWorld::instance()->fileName() || mX3dWorldGenerationTime != 0.0;
```

Comment:
`mX3dWorldReferenceFile` is probably useless now.

Reply:
You are right